gbi.h

```diff
@@ -252,10 +252,28 @@ longer a multiple of 8 (DMA word). This was not used in any command anyway. */
 #define G_MWO_aLIGHT_10          0x90
 #define G_MWO_bLIGHT_10          0x94
 
+/**
+ * @brief changes the color of the vertex. The val parameter is interpreted as 4 bytes: red (high byte), green, blue, and alpha (low byte).
+ * 
+ */
 #define G_MWO_POINT_RGBA         0x10
+/**
+ * @brief changes the S and T values (texture coordinates of the vertex). The high 16 bits of val specify the S coordinate, and the low 16 bits specify the T coordinate. Each coordinate is an S10.5 number.
+ * 
+ */
 #define G_MWO_POINT_ST           0x14
-#define G_MWO_POINT_XYSCREEN     0x18 /* not recommended to use, won't work if */
-#define G_MWO_POINT_ZSCREEN      0x1C /* the tri gets clipped */
+/**
+ * @brief change the screen coordinates of the vertex. The high 16 bits of val specify the X coordinate and the low 16 bits specify the Y coordinate. Both coordinates are S13.2 numbers with 0,0 being the upper-left of the screen, positive X going right, and positive Y going down.
+ * 
+ * not recommended to use, won't work if
```

Comment:
The end of this comment got moved to the next macro. It was originally a multiline comment across two definitions.